src/Security/Authentication/FrontendPreviewAuthenticator.php

```diff
@@ -89,30 +90,18 @@ public function authenticateFrontendUser(string $username): bool
             return false;
         }
 
-        try {
-            /** @var FrontendUser $frontendUser */
-            $frontendUser = $this->userProvider->loadUserByUsername($username);
-        } catch (UsernameNotFoundException $e) {
-            if (null !== $this->logger) {
-                $this->logger->info(
-                    sprintf('Could not find a front end user with the username "%s"', $username),
-                    ['contao' => new ContaoContext(__METHOD__, ContaoContext::ACCESS)]
-                );
-            }
-
-            return false;
-        }
+        if (null === $username) {
```

Comment:
I don't like this. Shouldn't this logic be in a separate method? It feels weird that the method requires a username but also accepts null.